### DISCUSSION TITLE
Tinymsc does not show in editor

### DISCUSSION THREAD

**Discussion by PH5HP** [e107inc/e107#5242] [General]
Hi,

Sorry for the typo in the message title.

I wonder if there is an option I turned off by accident, because when editing a page Tinymce doess not show up as editor. Just plain HTML input. I have another testwebsite and there it works as it should be.

I did compare some settings, but I still haven't found it if there is an option to turn on somewhere. Anyone who knows  the solution?

Kind regards,
Henk - ph5hp.nl - fileserver.ph5hp.nl

**Comment by Moc** [comment]
Did you check this preference? 

Admin Area > Preferences > Content Filters > Enable WYSIWYG textareas

Also, I assume the TinyMCE plugin is installed?

**Reply by PH5HP** [reply]
Hi,

TNX for trying to help me.

Yes, I have WYSISWYG enabled and the TinyMCE plugin has been installed.  The strange thing is that my testwebsite has the same preferences and there TinyMCE does work. All I get now is the plain HTML edit window.

Kind regards,
Henk - ph5hp.nl - fileserver.ph5hp.nl

**Reply by Alex-e107nl** [reply]
Hi Henk, you could try [https://github.com/Alex-e107nl/tinymce5](https://github.com/Alex-e107nl/tinymce5) uninstall version 4 and upload and install version 5.. 
If that does not work, what php version are you using?

**Reply by Moc** [reply]
@PH5HP Which e107 version are you using? Can you try uploading the latest stable files again (specifically the e107_plugin/tinymce4 folder)?

**Reply by PH5HP** [reply]
Hi,

TNX for trying to help me.

Alex:
 I will give that a try. I am using PHP version 7.4.33. It worked in the beginning. Also it still works at my test website which uses the same PHP and E107.

Moc:
I am using the lastest version 2.3.3. I did already re-uploaded the TinyMCE files. It did not change anything.

Kind regards,
Henk - ph5hp.nl - fileserver.ph5hp.nl

![image](https://github.com/e107inc/e107/assets/140949730/ac5e5818-278a-4cb8-b04b-55cb7a88b7c7)

**Comment by PH5HP** [comment]
Hi,

I did a little search and when I hit F12 in Google Chrome I get 'uncaught error'. See my screenshot below. I tried to copy a fresh TinyMCE to that plugin directory, but that did not help.

![image](https://github.com/e107inc/e107/assets/140949730/b7688e30-163b-440d-a2ce-6ebbfff320cc)

The second screenshot is where the link in the first screenshot brings me if I click on it.

![image](https://github.com/e107inc/e107/assets/140949730/83466d0a-21ca-40b7-bb3f-8f3200e125aa)

Kind regards,
Henk - ph5hp.nl - fileserver.ph5hp.nl

**Reply by tgtje** [reply]
just mentioning it : filters on or off ? (allowance?) often overseen..

**Reply by PH5HP** [reply]
Hi tgtje,

If you ment Preferences->content filters: they have the samme settings as on my test website where TinyMCE does works.

Kind regards,
Henk - ph5hp.nl - fileserver.ph5hp.nl

**Reply by Jimmi08** [reply]
That error from your images says something about emoticons, set it off, semicolor in in emoticon name causes an issue 

There was a way how to display wysiwyg directly to see errors better, but I don't know it and I don't have time to go through my old related issues.

**Reply by PH5HP** [reply]
Hi Jimmi08,

That solved the problem of TinyMCE also. My installed emoticons caused crashing TinyMCE. Problem solved.

Kind regards,
Henk - ph5hp.nl - fileserver.ph5hp.nl

**Reply by tgtje** [reply]
@PH5HP would you be so kind to close it (as solved/answered) if your happy with result (you are able to do so as original poster).
Thanks for the solution @Jimmi08 .

**Reply by PH5HP** [reply]
Done.
